src/admin-views/tribe-options-licenses.php

```diff
@@ -7,10 +7,29 @@
 	), Tribe__Main::$tec_url . 'license-keys/'
 );
 
-$link = esc_url( $link );
+$utm_link = esc_url( $link );
 
 // Explanatory text about license settings for the tab information box
-$html = __( '<p>The license key you received when completing your purchase from %1$s will grant you access to support and updates until it expires. You do not need to enter the key below for the plugins to work, but you will need to enter it to get automatic updates. <strong>Find your license keys at <a href="%2$s" target="_blank">%3$s</a></strong>.</p> <p>Each paid add-on has its own unique license key. Simply paste the key into its appropriate field on below, and give it a moment to validate. You know you\'re set when a green expiration date appears alongside a "valid" message.</p> <p>If you\'re seeing a red message telling you that your key isn\'t valid or is out of installs, visit <a href="%4$s" target="_blank">%5$s</a> to manage your installs or renew / upgrade your license.</p><p>Not seeing an update but expecting one? In WordPress, go to <a href="%6$s">Dashboard > Updates</a> and click "Check Again".</p>', 'tribe-common' );
+$html = '<p>' . sprintf(
+		esc_html__( 'The license key you received when completing your purchase from %1$s will grant you access to support and updates until it expires. You do not need to enter the key below for the plugins to work, but you will need to enter it to get automatic updates. %3$sFind your license keys at %2$s%4$s.', 'tribe-common' ),
+		'<a href="' . Tribe__Main::$tec_url . '" target="_blank">' . Tribe__Main::$tec_url . '</a>',
```

Comment:
I wonder if we should take this opportunity to add the `'<span class="screen-reader-text">' . __( ' (opens in new window)', 'tribe-common' ) . '</span>'` on all of these links with `target="_blank"` since they are being fiddled with now?